spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/core/DatastoreTemplateTests.java

```diff
@@ -318,47 +313,44 @@ public void performTransactionTest() {
 					return "all done";
 				});
 
-		assertEquals("all done", finalResult);
+		assertThat(finalResult).isEqualTo("all done");
 		verify(transactionContext, times(1)).put((FullEntity<?>) any());
 		verify(transactionContext, times(3)).get((Key[]) any());
 	}
 
 	@Test
 	public void findAllByIdTestNotNull() {
-		assertTrue(this.datastoreTemplate
-				.findAllById(Collections.singletonList(this.badKey), TestEntity.class)
-				.isEmpty());
+		assertThat(
+				this.datastoreTemplate.findAllById(Collections.singletonList(this.badKey), TestEntity.class)).isEmpty();
 	}
 
 	@Test
 	public void findByIdTest() {
 		TestEntity result = this.datastoreTemplate.findById(this.key1, TestEntity.class);
-		assertEquals(this.ob1, result);
-		assertThat(result.childEntities, contains(this.childEntity1));
-		assertEquals(result.singularReference, this.childEntity1);
-		assertThat(result.multipleReference, contains(this.childEntity1));
+		assertThat(result).isEqualTo(this.ob1);
+		assertThat(result.childEntities).contains(this.childEntity1);
+		assertThat(this.childEntity1).isEqualTo(result.singularReference);
+		assertThat(result.multipleReference).contains(this.childEntity1);
 	}
 
 	@Test
 	public void findByIdNotFoundTest() {
 		when(this.datastore.get(ArgumentMatchers.<Key[]>any())).thenReturn(null);
-		assertNull(
-				this.datastoreTemplate.findById(createFakeKey("key0"), TestEntity.class));
+		assertThat(this.datastoreTemplate.findById(createFakeKey("key0"), TestEntity.class)).isNull();
 	}
 
 	@Test
 	public void findAllByIdTest() {
 		when(this.datastore.get(eq(this.key1), eq(this.key2)))
 				.thenReturn(ImmutableList.of(this.e1, this.e2).iterator());
 		List<Key> keys = ImmutableList.of(this.key1, this.key2);
-		assertThat(this.datastoreTemplate.findAllById(keys, TestEntity.class),
-				contains(this.ob1, this.ob2));
+		assertThat(this.datastoreTemplate.findAllById(keys, TestEntity.class)).contains(this.ob1, this.ob2);
```

Comment:
Is this check meant to be ordered and/or precise? containsExactly() would enforce order and if-and-only-if.